chain/chain/src/chain.rs

```diff
@@ -361,7 +362,9 @@ impl Chain {
         let resharding_manager = ReshardingManager::new(
```

Comment:
Do we even need resharding manager in the view client?

Reply:
I don't see a reason. I'll make a note to ask @shreyan-gupta and then remove the manager from there

Reply:
Shouldn't be needed